src/commands/help.js

```diff
@@ -1,67 +1,70 @@
-const defaultHelp = `MMOくんはみんなでボスを倒して行くRPGです。
+const defaultHelp = {
+  embed: {
+    title: "Botの説明  :question:",
+    description: "MMOくんはみんなでボスを倒して行くRPGです。",
+    fields: [
+      { name: "help", value: "このメッセージを表示する" },
+      { name: "attack", value: "モンスターに攻撃する" },
+      { name: "status", value: "自分のステータスを確認する" },
+      { name: "inquiry", value: "チャンネルのバトルの状態を確認する" },
+      { name: "reset", value: "戦いをやり直す" },
+      { name: "t", value: "トレーニングをする" },
```

Comment:
一番初めに叩くことの多いコマンドなので、四字熟語とクイズの区別は付けた方がいいと思います。
```suggestion
      { name: "t", value: "四字熟語トレーニングをする" },
```